sgx/sync/src/condvar.rs

```diff
@@ -386,8 +393,8 @@ impl SgxCondvar {
     /// Wakes up one blocked thread on this condvar.
     ///
     /// If there is a blocked thread on this condition variable, then it will
-    /// be woken up from its call to [`wait`]. Calls to `signal` are not buffered
-    /// in any way.
+    /// be woken up from its call to [`wait`]. Calls to `signal` are not
+    /// buffered in any way.
     ///
     /// To wake up all threads, see [`broadcast`].
```

Comment:
@eranrund i think it may not be desirable to reformat this file, since we got it from baidu. i thought we had it in some rustfmt ignore group?

i'm okay either way i guess though

Reply:
It doesn't appear it is currently being ignored... https://github.com/mobilecoinfoundation/mobilecoin/blob/master/rustfmt.toml